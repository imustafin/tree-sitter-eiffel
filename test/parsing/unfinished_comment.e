-- no \n at the end of file