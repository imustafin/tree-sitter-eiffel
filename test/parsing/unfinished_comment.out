(source_file [0, 0] - [0, 27]
  (comment [0, 0] - [0, 27]))
